gaskij/rpicampusmap
# Set up Heroku hosting, automatic deploys, and CI testing pipelines

## Description
This PR includes the code to set up GitHub Actions to provide automatic testing, linting, and build checks when pushing code to any branch or creating a PR targeting master. This also sets up the basis for having automatic deployment to Heroku.

Fixes #116, fixes #123

## Solution
I created the `.github/workflows/main.yaml` file to set up GitHub Actions, and created the app and pipeline on Heroku to go along with it. The app can be found here: https://rpicampusmap.herokuapp.com/

## Known Issues
There are no issues caused by this PR, but there is a change to the linter. It now expects LF (Unix) linebreaks instead of CRLF (Windows).

## Testing Procedures
* Visit the deployed website and ensure it is running
* Become members of the team on Heroku and become familiar with the pipeline

## Checklist for author
